### PR TITLE
[Gabriela] Added IO task

### PR DESCRIPTION
**Pull request by gaenache**

### CODE REVIEW

File: gabrielaEnache/src/main/java/com/endava/io/NewIOUtils.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        }
+        return result;
+    }
+
+    public static Path createFile(String path, String filename) {
+        Path directory = Paths.get(path);
+        return directory.resolve(filename);
+    }
+
+    public static void writeInFile(String message, Path path) {
```
Comment (solonarieugeniu) [inline gabrielaEnache/src/main/java/com/endava/io/NewIOUtils.java]: you could try to read and write using FileChannel

**Review by solonarieugeniu — APPROVED** [review]

File: gabrielaEnache/src/main/java/com/endava/io/IOUtils.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.LineNumberReader;
+import java.util.ArrayList;
+import java.util.List;
+
+public class IOUtils {
```
Comment (solonarieugeniu) [inline gabrielaEnache/src/main/java/com/endava/io/IOUtils.java]: IOUtils is an appache class, please rename, e.g. FileUtils, NioFileUtils

**Review by asecu — COMMENTED** [review]
Remove please the 32k lines file

File: gabrielaEnache/src/main/java/com/endava/io/NewIOUtils.java (hunk 1 above)
Comment (gaenache) [reply gabrielaEnache/src/main/java/com/endava/io/NewIOUtils.java]: Thank you for suggestion. I will have a look.

File: gabrielaEnache/src/main/java/com/endava/io/IOUtils.java (hunk 2 above)
Comment (gaenache) [reply gabrielaEnache/src/main/java/com/endava/io/IOUtils.java]: Fixed

**Review by abadarau — APPROVED** [review]

**Review by asecu — APPROVED** [review]

**Pull request merged**